dsp-tools/dsp-atlantis/values.yaml

```diff
@@ -59,6 +59,11 @@ atlantis:
       secretKeyRef:
         name: atlantis-vault-approle
         key: secret-id
+    # legacy github account oauth token used by saturn
+    - name: TF_VAR_github_token
```

Comment:
I think it might be okay to rename this to `broadbot_deprecated_github_token` if you want (and just update the Saturn TF to use it), but no pressure!

Reply:
ended up not changing this as to not moving target too much. I have added a comment in the same line in terraform-ap-deployments so anyone who searches for github_token will also get the deprecation notice.

Reply:
i lied, i can't add the comment inline without terraform attempting to grok the line anew, and it's possssssible saturn is just _broken_ and can't run at all.